src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  person: string | null;
+  slug: string | null;
+  sex: boolean;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, slug, sex }) => {
+  if (!person) {
+    return '-';
+  }
```

Comment:
```suggestion
type Props = {
  person: Person;
};

export const PersonLink: React.FC<Props> = ({ person }) => {
```
You don't need to render the link without a person, and info such as slug/sex you get from this object